cmake/legacy.cmake

```diff
@@ -130,11 +130,17 @@ set(boost_features
   "visibility=hidden"
   "pch=off"
 )
+
+if(${Python_VERSION} VERSION_GREATER_EQUAL 3.13.0)
+  set(python_3_13_fix COMMAND ${patch} -p0 -i "${CMAKE_SOURCE_DIR}/legacy/boost/PyEval_CallMethod_fix_python_3-13.patch")
+endif()
+
```

Comment:
Is this patch really harmful on earlier Python releases?

I doubt that, as [`PyObject_CallMethod` exists for a long time now and was last changed in Python 3.4](https://docs.python.org/3/c-api/call.html#c.PyObject_CallMethod) (Debian-10 has 3.7).

If boost uses this patch conditionally, then I am fine.

Otherwise please patch unconditionally.

Reply:
Probably the patch isn't harmful for older versions of Python releases, but I didn't want to introduce new problems when trying to fix new ones such that I decide to do the patch conditionally.
I will check what is done by the boost project themself.

Reply:
Since boost itself uses the patch unconditionally I changed it in the same way.